specifyweb/specify/migrations/0010_cascading_tree_nodes.py

```diff
@@ -0,0 +1,49 @@
+# Generated by Django 3.2.15 on 2024-10-31 02:34
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
```

Comment:
TODO: 
Maybe change dependency once ready to merge

Reply:
💯